src/MockClassifier.Api/Controllers/InstitutionController.cs

```diff
@@ -1,12 +1,25 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using MockClassifier.Api.Interfaces;
 using MockClassifier.Api.Models;
+using MockClassifier.Api.Services.Dmr;
 
 namespace MockClassifier.Api.Controllers
 {
     [Route("input-from-dmr/institution")]
     [ApiController]
     public class InstitutionController : ControllerBase
     {
+        private readonly IDmrService _dmrService;
+        private readonly ITokenService _tokenService;
+        private readonly INaturalLanguageService _naturalLanguageService;
+
+        public InstitutionController(IDmrService dmrService,ITokenService tokenService, INaturalLanguageService naturalLanguageService)
```

Comment:
nit: I think code styling will want to see a space separating the parameters

```suggestion
        public InstitutionController(IDmrService dmrService, ITokenService tokenService, INaturalLanguageService naturalLanguageService)
```